docs/getting-started-from-release.md

```diff
@@ -0,0 +1,146 @@
+
+VDO operator is built to run on vanilla k8s cluster as well Openshift clusters
+
+With every new release of VDO, support for new features and various bug fixes are delivered
+
+## Assets
+
+Each release is launched with the following deliverables
+
+### VDO Docker Image
+
+Docker image for VDO can be found [here](https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/releases/download/0.1.1/vdo.tar.gz) 
+
+### VDO Deployment file
+
+[Deployment file](https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/releases/download/0.1.1/vdo-spec-vanilla-k8s.yaml) containing the specs of VDO which includes the CRD, Service accounts, Roles and Role Binding details 
+
+### Compatibility matrix
+
+To fetch the compatible CSI and CPI version according to the respective kubernetes/ vSphere versions, a new [Compatibility matrix](https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/releases/download/0.1.1/compatibility.yaml) yaml is released with every release 
+
+### Source Code 
+
+If you wish to Contribute or setup the Developer's environment, you can download our source code in [zip](https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/archive/refs/tags/0.1.1.zip) or [tar.gz](https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/releases/download/0.1.1/vdoctl-mac-0.1.1.tar.gz) format and for further steps, refer to the [Developer guide](developer-guide.md)  
+
+### VDOCTL
+
+In order to deploy VDO using the command-line interface VDOCTL, you can follow the below steps 
+
+#### Download the Binary
+
+Download the VDOCTL binary and untar it as
+
+- for Linux env
+```shell
+wget https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/releases/download/0.1.1/vdoctl-linux-0.1.1.tar.gz
+tar -xvf vdoctl-linux-0.1.1.tar.gz
+```
+
+- for Mac env
+```shell
+wget https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/releases/download/0.1.1/vdoctl-mac-0.1.1.tar.gz
+tar -xvf vdoctl-mac-0.1.1.tar.gz
+```
+
+Move the vdoctl executable to location in the `$PATH` folder
+
+#### Pre-requisite
+
+Before deploying the operator, set the KUBECONFIG path by
+```shell
+# Set up environment variables by
+export KUBECONFIG=~/.kube/config
+or YOUR-KUBECONFIG_PATH
+```
+
+#### Deploy VDO Operator
+
+Run the `vdoctl deploy` command to deploy VDO 
+
+```shell
+vdoctl deploy --spec https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/releases/download/0.1.1/vdo-spec-vanilla-k8s.yaml
+```
+Here, `spec` flag inputs the file containing details about the CRD, docker image
+
+For more details, please refer [vdo-spec-vanilla-k8s.yaml](https://github.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/releases/download/0.1.1/vdo-spec-vanilla-k8s.yaml)
+
+
+#### Configure Compatibility Matrix
+
+Once the VDO is deployed, you will see that the VDO manager pod is in `CreateConfigError` state and expects the compatibility matrix to be present
```

Comment:
This needs to change based on the latest code. we need to talk about the default compatibility matrix here.
we need to talk about where the default file is available and how they can configure the compatibility if required

Reply:
for each command lets ensure that we have line pointing to the command documentation

Reply:
as discussed, since we are deploying VDO with required pre-requisites we will not run into CreateConfigError